src/Symfony/Component/AssetMapper/ImportMap/ImportMapAuditor.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Symfony\Component\AssetMapper\ImportMap;
+
+use Symfony\Component\AssetMapper\Exception\RuntimeException;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class ImportMapAuditor
+{
+    private const AUDIT_URL = 'https://registry.npmjs.org/-/npm/v1/security/audits';
+
+    private readonly HttpClientInterface $httpClient;
+
+    public function __construct(
+        private readonly string $importMapConfigPath,
+        HttpClientInterface $httpClient = null,
+    ) {
+        $this->httpClient = $httpClient ?? HttpClient::create();
+    }
+
+    public function audit(): array
```

Comment:
it would be great to add phpdoc describing the structure of the returned array. This would make it easier to use this class by knowing what its signature actually is.
And it will also make it easier to maintain that class by knowing better which kind of changes would be BC breaks.

Reply:
It is now replaced by a `ImportMapPackageAudit` DTO.